docs/modules/ROOT/pages/faqs.adoc

```diff
@@ -1,83 +1,100 @@
 = Cassandra Source Connector FAQs
 
-If you are new to DataStax Cassandra Source Connector, these FAQs are for you.
+If you are new to DataStax Cassandra Source Connector (CSC), please read the following FAQs.
 
 == Introduction
 
-=== What is DataStax Cassandra Source Connector?
+=== What is DataStax CSC?
 
-The Alpha release of Cassandra Source Connector is a new, open-source product from DataStax.
-With CDC, Cassyour apps can consume data from an Apache Cassandra&reg; table and put the updates onto a Pulsar topic, which in turn can write the data to external targets such as Elastic, Snowflake, and other platforms. By design, the Source component is simple, with a 1:1 correspondence between the Cassandra table and a single Pulsar topic.
+The Alpha release of the Cassandra Source Connector (CSC) is a new, open-source product from DataStax.
 
-=== What are the system requirements for Cassandra Data Connector?
+With CDC, your apps can consume data from an Apache Cassandra table and submit the updates onto a Pulsar topic.  This topic can then be consumed by external targets such as Elastic, Snowflake, and other platforms. By design, the CSC is meant to be a simple connector with a 1:1 correspondence between a Cassandra table and a single Pulsar topic.
 
-Cassandra Source Connector requires additional resources depending on your write throughput.
-For each cassandra write, each Cassandra replica send one message to your streaming platform (Pulsar or Kafka), and
-execute at least one QUORUM read on your
+The CSC is composed of two parts:
 
-=== What components and features are provided by Cassandra Data Connector?
+* The Cassandra Source Connector, which transmits Change Data Capture (CDC) updates from the Cassandra table to the streaming service.
+* The Pulsar CSC Connector, which verifies the CSC events and stores the final data changes into a Data topic.
 
-Cassandra Data Connector provides:
+=== What are the system requirements for the CSC?
+
+The CSC requires additional resources depending on your write throughput.
+
+For each Cassandra write, each Cassandra replica sends one message to your Pulsar service, and executes at least one QUORUM read on your Cassandra cluster.
+
+=== What components and features are provided by Cassandra Source Connector?
+
+Cassandra Source Connector provides:
 
 * Cassandra, Elastic, Snowflake, Kinesis, Kafka, and JDBC connectors
 * A management dashboard
 * A monitoring and alerting system
 
-=== Which versions of Apache Pulsar does DataStax Cassandra Data Connector support?
+=== Which versions of Apache Pulsar does DataStax Cassandra Source Connector support?
 
-Cassandra Data Connector supports Apache Pulsar 2.7.0 and later.
+Cassandra Source Connector supports Apache Pulsar 2.7.0 and later.
```

Comment:
Shouldn't this be 2.8.0 now?

Reply:
We can also consider adding a single variable everywhere the Pulsar version number is listed.

Reply:
I'll have to look up how to do that with AsciiDoc, but that's a good idea.